src/widgets/torrent/TorrentTile.tsx

```diff
@@ -230,4 +237,21 @@ const filterTorrentsByLabels = (
   return torrents.filter((torrent) => !labels.includes(torrent.label as string));
 };
 
+export const getTorrentsRatio = (
+  widget: ITorrent,
+  torrents: NormalizedTorrent[],
+  applyAllFilter:boolean
+) => {
+
+  if(applyAllFilter) {
+    torrents = filterTorrents(widget,torrents)
+  } else if (widget.properties.labelFilter.length > 0) {
+    torrents = filterTorrentsByLabels(torrents, widget.properties.labelFilter,widget.properties.labelFilterIsWhitelist)
+  }
+
+  let totalDownloadedSum = torrents.reduce((sum, torrent) => sum + torrent.totalDownloaded, 0);
+
+  return totalDownloadedSum > 0 ? torrents.reduce((sum, torrent) => sum + torrent.totalUploaded, 0) / totalDownloadedSum : -1;
+}
```

Comment:
Same for this function.
Just go into vs code, press f1, Format Document.
The code itself is fine, it's just for readability.
I suggest you do it on all the code files you modify.

Reply:
Done